#!/usr/bin/env python
PACKAGE = "automodel_line_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("canny_lowThreshold", int_t, 0, "Low threshold for Canny", 165, 0, 255)
gen.add("canny_highThreshold", int_t, 0, "High threshold for Canny", 222, 0, 255)
gen.add("canny_perBlindHorizon", int_t, 0, "Percentage for blind horizon", 46, 0, 100)
gen.add("hough_int_rho", int_t, 0, "Hough Rho", 1, 1, 50)
gen.add("hough_int_theta", int_t, 0, "Hough Theta", 1, 1, 360)
gen.add("hough_threshold", int_t, 0, "Hough Threshold", 45, 1, 300)

exit(gen.generate(PACKAGE, "automodel_line_detector", "line_detector"))
